libs/shared/src/config/trigger-workflow.ts

```diff
@@ -0,0 +1,20 @@
+export enum QueueNameEnum {
+  INBOUND_PARSE_MAIL = 'inbound-parse-mail',
+  METRICS = 'metric',
+  STANDARD = 'standard',
+  WEB_SOCKETS = 'ws_socket_queue',
+  WORKFLOW = 'trigger-handler',
+}
+
+export enum WorkerNameEnum {
+  INBOUND_PARSE_MAIL = 'inbound-parse-mail',
+  METRICS = 'metric',
+  STANDARD = 'standard',
+  WEB_SOCKETS = 'ws_socket_queue',
+  WORKFLOW = 'trigger-handler',
+}
```

Comment:
_Duplicating_ them as I want to split in the future the Queues and Workers to not be mixed up like it happens in some classes that extend from Queue services and then instantiate Workers.

Reply:
I'm not sure about having the separate enums, because the worker takes the queue name when you create it.
Now I would have to remember to call the queue using one enum and to call workers using another enum that should have the exact same value. Why complicate our lives, what is the benefit?

Reply:
Separation of concerns is the benefit. It is an extra help to understand what's going on when using it.
If you are instantiating a worker using for example a reference to a queue name might mislead you to think that's a queue, while they are different things. For me is one of the problems we have had in our current codebase due to naming and mixing concerns and want to slowly fix it up.

Reply:
You know that they are "coupled" by the queue name. But if the reference name is a problem here and misleading let's maybe not use the enums but instead just the constants?

Reply:
As I finally separated the Queues and Workers in the new implementation here: https://github.com/novuhq/novu/pull/3916
having the naming separated to distinguish doesn't bother me anymore and I will use a single reference. Just will rename it as `JobTopicNameEnum` as basically is what it refers, the name of the topic where the jobs are published and then consumed from. Topic is the name used across the terminology of majority of the message brokers. 👍🏻